app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,184 @@
+    background-size: 70%;
+    padding-bottom: 0;
+  }
+
+  @include respond-to($mq-l) {
+    background-position: 141% 100%;
+    background-size: 65%;
+  }
+
+  @include respond-to($mq-xl) {
+    background: none;
+    overflow: hidden;
+  }
+}
+
+.home-top-trust {
+  position: relative;
+
+  @include respond-to($mq-xl) {
+    position: absolute;
+    top: 0;
+    width: 1000px;
+  }
+}
+
+.home-top__image {
+  @include respond-to($mq-xl) {
+    background: image-url("illustrated_people.png") 100% 52% no-repeat;
+    background-size: 49%;
+    height: 309px;
+    margin: 0 auto;
+    position: relative;
+    width: 1350px;
+
+    .theme-cy & {
+      background-position: 100% 100%;
+      background-size: 51%;
+      height: 351px;
+    }
+  }
+}
+
+.home-top-trust__heading {
+  @extend %font-heading-heavy;
+  @include body(36, 42);
+  color: $color-heading-extra-small;
+  letter-spacing: -1.5px;
+  margin: 0;
+  padding-bottom: $baseline-unit*5;
+  padding-top: $baseline-unit*5;
+
+  @include respond-to(px(600), px(719)) {
+    max-width: 500px;
+  }
+
+  @include respond-to($mq-m) {
+    @include body(48, 54);
+
+    .theme-cy & {
+      max-width: 600px;
+    }
+  }
+
+  @include respond-to($mq-l) {
+    max-width: 530px;
+
+    .theme-cy & {
+      max-width: 500px;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    max-width: 700px;
+
+    .theme-cy & {
+      max-width: 750px;
+    }
+  }
+}
+
+.home-top-trust-content {
+  @extend .unstyled-list;
+  @extend .list-benefits;
+  margin-bottom: $baseline-unit*2;
+}
+
+.home-top-trust-content__item {
+  @extend %font-heading-medium;
+  @include body(16, 24);
+  margin-bottom: $baseline-unit;
+
+  @include respond-to($mq-m) {
+    max-width: 45%;
+  }
+
+  @include respond-to($mq-l) {
+    max-width: none;
+
+    .theme-cy & {
+      max-width: 570px;
+    }
+  }
+
+  @include respond-to($mq-l, $mq-xl) {
+    .theme-cy & {
+      max-width: 50%;
+    }
+  }
+}
+
+.home-top-promo {
+  clear: both;
+
+  @include respond-to(px(600), px(719)) {
+    bottom: 0;
+    left: 0;
+    position: absolute;
+  }
+
+  @include respond-to($mq-l) {
+    bottom: 0;
+    clear: none;
+    margin: 0;
+    position: absolute;
+    right: 3%;
+  }
+}
+
+.home-top-promo__text {
+  @extend %font-heading-heavy;
+  @include body(22, 30);
+  background: #000;
+  border-radius: 6px;
+  margin: 0 $baseline-unit*3;
+  max-width: 400px;
+  padding: $baseline-unit*4 $baseline-unit*7 $baseline-unit*4 $baseline-unit*4;
+  position: relative;
+
+  @include respond-to(px(600)) {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  @include respond-to($mq-m) {
+    margin: $baseline-unit*2 0 0 0;
+    padding: $baseline-unit*2 $baseline-unit*8 $baseline-unit*2 $baseline-unit*4;
+    width: 55%;
+  }
+
+  @include respond-to($mq-l) {
+    width: 80%;
+  }
+}
+
+.home-top-promo-link {
+  &:link,
+  &:focus,
+  &:visited,
+  &:hover {
+    color: $color-white;
+  }
+}
+
+.home-top-promo-link__arrow {
```

Comment:
Selector `home-top-promo-link__arrow` should be written in lowercase with hyphens